### ISSUE TITLE
Unhashable type _bridged_ghidra.program.database.function.FunctionDB

### ISSUE THREAD

**Issue by 6r0k3d** [justfoxing/ghidra_bridge#63]
I'm moving from a python ghidra_script to using ghidra_bridge, in the script theres a lookup to see if a function was saved to a cache:

`if func not in highfunction_cache:	`

But I'm getting the following error
```
    if func not in highfunction_cache:
TypeError: unhashable type: '_bridged_ghidra.program.database.function.FunctionDB'
```

It looks like func is now type `<class 'jfx_bridge.bridge._bridged_ghidra.program.database.function.FunctionDB'>`

Is there a way to get the underlying FunctionDB type that can then be looked up in the dict?

**Comment by justfoxing** [comment]
Fixed in jfx-bridge 0.8.1 (jfx-bridge provides the bridging technology for ghidra_bridge) - the __hash__() function wasn't being proxied through, making the bridged objects unhashable, and preventing them from being inserted as dictionary keys.

Upgrade with `pip install ghidra_bridge --upgrade --force-reinstall` to get the latest jfx_bridge component, then re-install the server scripts with `python -m ghidra_bridge.install_server <script location>`.

**Comment by 6r0k3d** [comment]
Hey! That was fast! Fixed the hashing, running into another problem though, submitting an issue now
